lambdas/indexer/app.py

```diff
@@ -115,12 +151,7 @@ def update_health_cache(_event: chalice.app.CloudWatchEvent):
     app.health_controller.update_cache()
 
 
-@app.route('/', cors=True)
-def hello():
-    return {'Hello': 'World!'}
-
-
-@app.route('/{catalog}/{action}', methods=['POST'])
+@app.route('/{catalog}/{action}', methods=['POST'], interactive=False)
```

Comment:
Please explain the need for `interactive=False`

Reply:
Because this route requires HMAC authentication, it will fail with a 401 error if a user tries to use the "Try it out" button in the Swagger UI. Setting `interactive=False` disables the "Try it out" button.

Reply:
This change could be postponed until https://github.com/DataBiosphere/azul/issues/5051 since currently none of the routes define any actual spec, so the "Try it out" button wouldn't be visible anyway.

Reply:
Decided at PL to postpone this change until #5051